### DISCUSSION TITLE
Multiple commands

### DISCUSSION THREAD

**Discussion by omnigenous** [davmac314/dinit#410] [Q&A] [answered]
```
command = echo hello & sleep 3 && echo world
```

logs: hello & sleep 3 && echo world

```
command = sh -c 'echo hello & sleep 3 && echo world'
```

logs: hello: -c: line 1: unexpected EOF while looking for matching `''

Is there a proper way to do this without creating separate script file?

I basically need to run another command once daemon is started (first command)

**Comment by davmac314** [comment, marked as answer]
Use double quotes (") not single quotes ('). Single quotes are meaningless to dinit; see the documentation.
```
command = sh -c "echo hello & sleep 3 && echo world"
```

**Reply by omnigenous** [reply]
That did it! Didn't think about quotes too much. Thank you!

**Comment by nnsite** [comment]
What if we could pile multiple "command" and "stop-command" lines with scripted services ? They don't have to keep track of PID, all it would do would be chaining commands. It would avoid the ugly of sight of a shell call. And would provide much appreciated symmetry with directives allowing for multiple lines.
